docs/introduction/terminology.md

```diff
@@ -3,100 +3,93 @@ id: terminology
 
-:::info
-Currently, as of version 0.5.2, you must run one instance of Bee-clef for every instance of Bee.
-:::
+### Bee
 
-- [Install Bee-clef](/docs/installation/bee-clef)
+Swarm nodes are referred to as "Bee" nodes. Bee nodes can run on a wide variety of computer types including desktop computers, hobbyist computers like Raspberry Pis, remotely hosted virtual machines, and much more. When running, Bee nodes interact with Swarm smart contracts on Gnosis Chain and connect with other Bee nodes to form the Swarm network.
 
-## Swarm
+Bee nodes can act as both client and service provider, or solely as client or service provider, depending on the needs of the node operator. Bee nodes pay each other for services on the Swarm network with the xBZZ token.
 
-Swarm is the collective term for all of the Bee nodes within the network. It is the system that, together, provides the distributed and decentralised storage and communication system accomplishing the overarching goal.
 
-Swarm is also the name of the core organization that oversees the development and success of the Bee Swarm as a whole. You can find them at [swarm.ethereum.org](https://swarm.ethereum.org/).
+### Overlay and Underlay
 
-## Ethereum / Blockchain
+An overlay network is a virtual or logical network built on top of some lower level "underlay" network. Examples include the Internet as an overlay network built on top of the telephone network, and the p2p Bittorent network built on top of the Internet. 
 
-Ethereum is most commonly known by its traded coin, ETH, but in actuality, the Ethereum blockchain framework provides other foundational services upon which makes the higher-level goals of Bee possible.
+With Swarm, the overlay network is a [Kademlia DHT](https://en.wikipedia.org/wiki/Kademlia) with overlay addresses taken from each node's [Gnosis](terminology#gnosis-chain) address. Swarm's overlay network addresses are permanent identifiers for each node and do not change over time.
 
-Ethereum provides a service known as "smart contracts" which, in basic terms, is essentially the ability to publish rules or a binding digital code-of-conduct that establish the proper flow and exchange of data. Bee uses these contracts for two main purposes: To establish the Swarm incentivization token "BZZ", and to establish the concepts of Cheques and ChequeBooks.
+An underlay network is the low level network on which an overlay network is built. It allows nodes to find each other, communicate, and transfer data. Swarm's underlay network is a p2p network built with [libp2p](https://libp2p.io/). Underlay address are not permanent and may change over time. Other networks may in theory be used as the underlay as long as they meet the Swarm protocol specifications, however at this time libp2p is the only implementation of the Swarm underlay network.
```

Comment:
not sure about this last statement - i can't perceive a topology with multiple underlay networks unless all nodes ran both transports. perhaps one for discussion 😄

Reply:
Ah yeah I think I didn't phrase that properly, I didn't mean multiple networks, but multiple network protocols. I was referring to the Book of Swarm paper where it says "in its overall design, Swarm is agnostic of the particular underlay transport used as long as it satisfies the following requirements." I'll revise this part.